tests/shared/pages/InboxPage.ts

```diff
@@ -40,21 +47,74 @@ export class ModalPage {
     await this.page.locator('.NotificationPwaModal__close-button').first().click()
   }
 
+  async getAddress() {
+    await this.page.locator('.Avatar').first().click()
+    const address = await this.page.locator('wui-avatar').getAttribute('alt')
+    await this.page.locator('wui-icon[name=close]').first().click();
+
+    return address;
+  }
+
   async subscribe(nth: number) {
-    await this.page.locator('.AppCard__body > .AppCard__body__subscribe').nth(nth).click()
-    await this.page.getByText('Subscribed to', { exact: false }).isVisible()
+    const appCard = this.page.locator('.AppCard__body').nth(nth)
+    await appCard.locator('.AppCard__body__subscribe').click()
+
+    await appCard.locator('.AppCard__body__subscribed').getByText('Subscribed', { exact: false }).isVisible()
   }
 
-  async unsubscribe(nth: number) {
+  async navigateToNewSubscription(nth: number) {
     await this.page.getByRole('button', { name: 'Subscribed' }).nth(nth).click()
     await this.page.getByRole('button', { name: 'Subscribed' }).nth(nth).isHidden()
+  }
+
+  async subscribeAndNavigateToDapp(nth: number) {
+    await this.subscribe(nth);
+    await this.navigateToNewSubscription(nth);
+  }
+
+  async unsubscribe() {
     await this.page.locator('.AppNotificationsHeader__wrapper > .Dropdown').click()
     await this.page.getByRole('button', { name: 'Unsubscribe' }).click()
     await this.page.getByRole('button', { name: 'Unsubscribe' }).nth(1).click()
     await this.page.getByText('Unsubscribed from', { exact: false }).isVisible()
     await this.page.waitForTimeout(2000)
   }
 
+  async navigateToDappFromSidebar(nth: number) {
+    await this.page.locator('.AppSelector__notifications-link').nth(nth).click()
+  }
+
+  async countSubscribedDapps() {
+
+    const selected = await this.page.locator('AppSelector__list').all()
+    return (await this.page.locator('.AppSelector__notifications').count() - 1)
```

Comment:
Can we make this part more readable?
```suggestion
    const notificationsCount = await this.page.locator('.AppSelector__notifications').count()
    
    return notificationsCount - 1;
```